src/ZfcDatagrid/Datagrid.php

```diff
@@ -749,11 +760,7 @@ public function getRowStyles()
      */
     public function hasRowStyles()
     {
-        if (count($this->rowStyles) > 0) {
-            return true;
-        }
-
-        return false;
+        return (bool) $this->rowStyles;
```

Comment:
@kokspflanze this is shorter, but i (currently/personally) dont like it.
It's harder to understand IMO (someone who comes from another language wont know such a special feature).

But no worry, i'll merge it still...it seems to work at least